src/includes/go.inc.js

```diff
@@ -360,6 +360,10 @@ function drupalgap_back() {
       });
     }
     else if (active_page_id == '_drupalgap_splash') { return; }
+    else if (drupalgap.back_path == ''){
```

Comment:
If I add ` _drupalgap_back();` to line 364, the app flashes to the home page and then returns to the current page. Maybe this solution only works for web apps, and isn't ideal for a native app? 

I could only see needing to address this in a native app if a notification required routing to an internal page, though I imagine it would be handled much differently at that point since the desired page would be passed by the notification data as opposed to the URL. I'll go ahead and close this pull request and add a note to the docs for others that might find this edge case useful.